src/SearchResults.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Moment from "react-moment";
+import TableRow from "./TableRow";
+
+const SearchResults = props => {
+  console.log(props);
```

Comment:
Hey @zkhing, be sure to remove any rogue `console.log`s from your code - they should only be their for debugging purposes. Your production code shouldn't contain any logs so it's good to get into the habit early on.